src/content/certifications/exam-faqs/consul-associate-002.mdx

```diff
@@ -0,0 +1,100 @@
+<!-- Each "## Heading Two" section in this document will be used to populate an FAQ item -->
```

Comment:
Interesting! Is this how it's set up on .com as well? I see now why we would want to add headings via html here with this setup.

Reply:
Not necessary for this batch of work but as a post-launch follow-up it'd be nice to have some authoring notes / guidelines. Perhaps a README at `content/certifications`?

Reply:
Sweet! I've just pushed up [content/certifications/README.md](https://github.com/hashicorp/dev-portal/blob/bd45acc953a260d56393323413b1248f85d4fbeb/src/content/certifications/README.md), and have linked to it from the main [content/README.md](https://github.com/hashicorp/dev-portal/blob/bd45acc953a260d56393323413b1248f85d4fbeb/src/content/README.md) in a new Certifications section.